### PR TITLE
Rationalized OI button mapping.

### PR DESCRIPTION
**Pull request by SethNguyenXBOT**

### CODE REVIEW

File: Competition2017/src/competition/operator_interface/OperatorCommandMap.java [hunk 1]
```diff
@@ -100,9 +100,9 @@ public void setupShooterBeltCommands(
             ShooterBeltsManagerSubsystem shooterBeltsSubsystem)
     {
         RunShooterBeltPowerCommand runBeltLeft = new RunShooterBeltPowerCommand(shooterBeltsSubsystem.getLeftBelt());
-        oi.leftButtons.getifAvailable(2).whileHeld(runBeltLeft);
+        oi.controller.getXboxButton(XboxButton.LeftStick).whileHeld(runBeltLeft);
```
Comment (aschokking) [inline Competition2017/src/competition/operator_interface/OperatorCommandMap.java]: So to fire the shooter someone will hold down on a given side: Stick, Bumper and Trigger?
I'm wondering if we can simplify that down a bit now that we have a mostly functioning robot, for instance can we have 1 button that spins up the shooter and runs the belt while at speed?

**Review by JohnGilb — CHANGES_REQUESTED** [review]
We need the commands that set the shooters to speed goals to be on the gamepad. Right now they are on the joysticks.

**Comment by WasabiFan** [comment]
Remember that you have the D-pad to work with too... that might make more sense for ascend/descend.

**Comment by aschokking** [comment]
unless something has changed, the last time it was tested I believe the dpad wasn't functioning

**Comment by WasabiFan** [comment]
How so? The interface was implemented, so if there's a bug preventing it from getting data, that should be investigated.

**Review by JohnGilb — APPROVED** [review]

**Review by aschokking — APPROVED** [review]

**Comment by aschokking** [comment]
we should write this mapping down somewhere that we can keep near the OI

**Pull request merged**